src/App.tsx

```diff
@@ -1,18 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const [total] = useState(42);
```

Comment:
Why did you wrap this number to useState?
Let's use 
```
const items = getNumbers(1, 42)
  .map(n => `Item ${n}`);
```

You can get total from `items.length`